mappings/net/minecraft/entity/EntityGroup.mapping

```diff
@@ -0,0 +1,6 @@
+CLASS ahi net/minecraft/entity/EntityGroup
+	FIELD a DEFAULT Lahi;
```

Comment:
`NONE`?

Reply:
I don't think none would be appropriate here, since it's definitely a group still, it's just very broad.